### ISSUE TITLE
[ enhancement ] Marigold depth estimator 

### ISSUE THREAD

**Issue by Pirog17000** [Acly/krita-ai-diffusion#244]
Would be nice to have a few options to select from for the depth estimation algorithms, specifically new Marigold as it catches a LOT of details
https://huggingface.co/spaces/toshas/marigold

Additionally, Zoe if possible. Current depth estimator is fast but outputs are far worse than they could be :/ 

And again let me tell that the plugin is a best thing right now for diffusion, thank you for this!

**Comment by Acly** [comment]
I'm having a look at https://github.com/kijai/ComfyUI-Marigold every now and then. It's still a bit rough around the edges, also it's _very_ heavy weight (download size, vram reqs, execution speed). Results are nice though...

**Comment by ExecratedEdits** [comment]
Apologies if this belongs in a separate issue, but if Marigold isn't currently an option, are other depth model options still being considered? LeReS with boost enabled and matching the resolution to the image height also yields really good results, already comes with stock ComfyUI, and defaults to a tiling method so VRAM shouldn't be a problem.

**Comment by Acly** [comment]
Changed to depth-anything in the last update. It's similar to the one before though, better in some cases.

I tested LeReS too, it catches a LOT more detail, the depth maps look pretty good. In my scenarios it really didn't translate to good results with depth CN though, images contained a lot of non-sensical details and garbage that would have to be cleaned up. They didn't match the input image well at all.

I'm sure it highly depends on the scenario, but it didn't seem like a good general purpose solution.
